src/axom/inlet/SphinxDocWriter.hpp

```diff
@@ -145,14 +145,27 @@ class SphinxDocWriter : public DocWriter {
     *******************************************************************************
     */
     TableData() {
-      rstTable = {{"Field Name", "Description", "Default Value", "Range", "Required"}};
+      rstTable = {{"Field Name", "Description", "Default Value", 
+                             "Range/Valid Values", "Required"}};
     }
 
     std::string tableName;
     std::string description;
     std::vector<std::vector<std::string>> rstTable;
   };
 
+  /*!
+    *******************************************************************************
+    * \brief Stores all field info into m_rstTables

```

Comment:
Could you please improve the documentation for this function?
After reading this a few times, it is not entirely clear to me what this does.

Perhaps the function should be renamed to something like `extractFieldMetadata`?

Reply:
Also, the docs should not refer to implementation details like `m_rstTables`.

Reply:
I mentioned m_rstTables since the function is private--does that change anything?

Reply:
m_rstTables doesn't tell you anything about how or why you are storing the data. It also doesn't say what you are doing with the information.

Reply:
Something like "This extracts information about the Field stored in the given Sidre Group. This information is stored internally by the class and then written out by writeAllTables"